src/drivers/imx/sdma.c

```diff
@@ -159,6 +159,8 @@ static int sdma_register_init(struct dma *dma)
 		return ret;
 	}
 
+	dma_reg_write(dma, SDMA_MC0PTR, 0);
+
```

Comment:
Wait, when is it written?

Reply:
@paulstelian97 it is written later in the same function
```
»       dma_reg_write(dma, SDMA_MC0PTR, (uint32_t)pdata->ccb_array);
```
Also, the reset value should already be 0 but I've seen Linux driver doing this so for consistency I am also doing it here.